src/main/java/org/apache/maven/plugin/compiler/CompilerMojo.java

```diff
@@ -231,6 +231,7 @@ protected void preparePaths( Set<File> sourceFiles )
                 
                 ResolvePathsRequest<File> request =
                     ResolvePathsRequest.ofFiles( dependencyArtifacts )
+                                       .setIncludeStatic( true )
```

Comment:
I don't think we need this one.

Reply:
in the case of https://issues.apache.org/jira/browse/MCOMPILER-481 we do.